Cannot create/open project on Windows

Trying to run 0.20.0 on Windows without any cmd arguments. The project selection dialog appears, but after selecting "No project" or "Specify project" and giving it an existing directory, the following error occurs:

```
C:\>paintera
Exception in Application start method
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:389)
        at com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:328)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at sun.launcher.LauncherHelper$FXHelper.main(Unknown Source)
Caused by: java.lang.RuntimeException: Exception in Application start method
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$1(LauncherImpl.java:182)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ExceptionInInitializerError
        at org.janelia.saalfeldlab.paintera.config.input.KeyAndMouseConfig.<init>(KeyAndMouseConfig.kt:12)
        at org.janelia.saalfeldlab.paintera.Paintera.startImpl(Paintera.java:156)
        at org.janelia.saalfeldlab.paintera.Paintera.start(Paintera.java:88)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$8(LauncherImpl.java:863)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$7(PlatformImpl.java:326)
        at com.sun.javafx.application.PlatformImpl.lambda$null$5(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$6(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$null$3(WinApplication.java:177)
        ... 1 more
Caused by: java.util.regex.PatternSyntaxException: Illegal/unsupported escape sequence near index 4
^C:\Users\username
    ^
        at java.util.regex.Pattern.error(Unknown Source)
        at java.util.regex.Pattern.escape(Unknown Source)
        at java.util.regex.Pattern.atom(Unknown Source)
        at java.util.regex.Pattern.sequence(Unknown Source)
        at java.util.regex.Pattern.expr(Unknown Source)
        at java.util.regex.Pattern.compile(Unknown Source)
        at java.util.regex.Pattern.<init>(Unknown Source)
        at java.util.regex.Pattern.compile(Unknown Source)
        at kotlin.text.Regex.<init>(Regex.kt:89)
        at org.janelia.saalfeldlab.paintera.PainteraMainWindow.<clinit>(PainteraMainWindow.kt:351)
        ... 12 more
```

The lock file is created as usual, and if restarting the application with the same project directory, it detects the lock file asks to remove it. After that, the above error occurs again.

--- a/src/main/java/org/janelia/saalfeldlab/paintera/PainteraMainWindow.kt
+++ b/src/main/java/org/janelia/saalfeldlab/paintera/PainteraMainWindow.kt
@@ -264,7 +264,7 @@ class PainteraMainWindow(val gateway: PainteraGateway = PainteraGateway()) {
 
 	fun setupStage(stage: Stage) {
 		keyTracker.installInto(stage)
-		projectDirectory.addListener { pd -> stage.title = if (pd.directory == null) NAME else "$NAME ${replaceUserHomeWithTilde(pd.directory.absolutePath)}" }
+		projectDirectory.addListener { pd -> stage.title = if (pd.directory == null) NAME else "$NAME ${pd.directory.absolutePath.homeToTilde()}" }
 		stage.icons.addAll(
 				Image(javaClass.getResourceAsStream("/icon-16.png")),
 				Image(javaClass.getResourceAsStream("/icon-32.png")),
@@ -348,15 +348,14 @@ class PainteraMainWindow(val gateway: PainteraGateway = PainteraGateway()) {
 
 		private val USER_HOME = System.getProperty("user.home")
 
-		private val USER_HOME_AT_BEGINNING_REGEX = "^$USER_HOME".toRegex()
+		private const val TILDE = "~"
 
-		private val TILDE_AT_BEGINNING_REGEX = "^~".toRegex()
-
-		private fun replaceUserHomeWithTilde(path: String) = path.replaceFirst(USER_HOME_AT_BEGINNING_REGEX, "~")
+		private fun replaceUserHomeWithTilde(path: String) = if (path.startsWith(USER_HOME)) path.replaceFirst(USER_HOME, TILDE) else path
 
 		private fun String.homeToTilde() = replaceUserHomeWithTilde(this)
 
-		private fun String.tildeToHome() = this.replaceFirst(TILDE_AT_BEGINNING_REGEX, USER_HOME)
+		private fun String.tildeToHome() = if (this.startsWith(TILDE)) this.replaceFirst(TILDE, USER_HOME) else this
+
 		private val NAMED_COMBINATIONS = NamedKeyCombination.CombinationMap(
 				NamedKeyCombination("open data", KeyCodeCombination(KeyCode.O, KeyCombination.CONTROL_DOWN)),
 				NamedKeyCombination("save", KeyCodeCombination(KeyCode.S, KeyCombination.CONTROL_DOWN)),

